src/persist-client/src/impl/machine.rs

```diff
@@ -169,9 +169,11 @@ where
                 } else {
                     Duration::from_secs(1)
                 };
-                info!(
+                trace!(
```

Comment:
hmmm, i'm not sure I agree with this. it's pretty interesting when things retry and it shouldn't be spammy unless we're using unreliable. i'd prefer to keep these at info and re-evaluate when we have data from production logs (or something closer to them)